src/app.js

```diff
@@ -10,6 +12,8 @@ app.use(logger(formatsLogger));
 app.use(cors());
 app.use(express.json());
 
+app.use('/appointments', routerAppointments);
```

Comment:
Lets use `/api/appointments` route

Reply:
Agree, but don't think using "/api/" before every router, I'll try to find a better way to use it.